client/src/legacy/TinyMCE_sslink.js

```diff
@@ -111,7 +111,53 @@ jQuery.entwine('ss', ($) => {
       /* noop */
     },
 
-  /**
+    /**
+     * @param selection The current tinyMCE selection for this WYSIWYG
+     * @return {Boolean}
+     */
+    linkCanWrapSelection(selection) {
+      const selectionContent = selection.getContent() || '';
+      const node = selection.getNode();
+
+      // If there is direct selected content other than whitespace, we can wrap it.
+      if (selectionContent) {
+        return selectionContent.trim() !== '';
+      }
+
+      // If the selected node type can contain text, and we didn't find selected text above,
+      // then you haven't got a selection we can wrap in a link.
+      const x = document.createElement(node.nodeName);
```

Comment:
What is the point of this? Seems like it will will always return false since we're just testing an element we're creating ourselves?  Or is there some sort of difference between browsers?

Reply:
We're essentially getting the browser to check for us if this node _can_ have text. If it can have text, and we didn't have any text selected, then this selection isn't valid for wrapping a link, since we'd want the link to wrap the text.

If this element _cannot_ have text, then we go into the next set of conditional statements to see if this is an element that can be wrapped inside a link (like img), or if it's something that cannot (like iframe or another link)

Reply:
OK sorry I totally missed the bit where `node` is variable :)